appsec/src/helper/subscriber/waf.cpp

```diff
@@ -419,6 +430,18 @@ void instance::listener::submit_metrics(
             msubmitter.submit_span_metric(
                 metrics::rasp_timeout, rasp_timeouts_);
         }
+
+        for (auto const &rule : rasp_metrics_) {
+            metrics::telemetry_tags tags;
+            tags.add("rule_type", rule.first);
+            tags.add("waf_version", ddwaf_get_version());
```

Comment:
There are two other tags (newly defined) that are missing:
- `event_rules_version`
- `block` (only for `rasp.rule.match`)